### ISSUE TITLE
Redo assets

### ISSUE THREAD

**Issue by Bubu** [Bubu/android-dice-game#3]
This will have a number of benefits:

* Don't use the original catan artwork
* Higher resolution -> scales better to higher resolutions
* Make it translatable.

**Comment by lramage94** [comment]
It would be great if you could reach out on https://opengameart.org/ for an opportunity to collaborate on this.

**Comment by Bubu** [comment]
@lramage94 Good idea. I won't have the time to lead that effort though, maybe someone else could do that?
